cudf/tests/test_csvreader.py

```diff
@@ -144,3 +144,26 @@ def test_csv_reader_skiprows_skipfooter(tmpdir):
     assert len(out.columns) == len(df_out.columns)
     assert len(out) == len(df_out)
     pd.util.testing.assert_frame_equal(df_out, out.to_pandas())
+
+
+def test_csv_reader_negative_vals(tmpdir):
+    fname = 'tmp_csvreader_file8.csv'
```

Comment:
Can you follow the way we specify filename / path in the previous tests?

Reply:
done in latest commit